### PR TITLE
Redirect loggedin user

### PR DESCRIPTION
**Pull request by PatriciaRosalis**
**TASK**:
- if a user goes to /auth/login, check they are logged in.
- if they are logged in redirect them to the /courses page.

### CODE REVIEW

**Comment by eleanorreem** [comment]
Hi Rosalia, Thank you so much for your PR. Your solution works well.
I would suggest one improvement - copy the code snippet you wrote in the NavigationMenu file in /pages/auth/login.tsx. Navigation Menu loads on every page and this snippet isn't necessarily relevant to every page. If you put it in  /pages/auth/login.tsx, it would only run on the /auth/login page and you could remove the later part of your if statement checking the url. I hope that makes sense. Let me know if you have any questions!

File: components/layout/NavigationMenu.tsx [hunk 1]
```diff
@@ -113,7 +119,7 @@ const NavigationMenu = (props: NavigationMenuProps) => {
     }
 
     setNavigationLinks(links);
-  }, [partnerAccesses, t, user, partnerAdmin]);
+  }, [partnerAccesses, t, user, partnerAdmin, router]);
```
Comment (swetha-charles) [inline components/layout/NavigationMenu.tsx:122]: Thanks so much for the PR! 

Echoing what @eleanorreem said, you could add a useEffect in the Login page with the same code snippet. 

We only want to run the effect when the user changes, so just having the user in the dependency array would be better. The router might change for other reasons - we don't necessarily need to run the effect if that happens.  

Let me know if that makes sense :)

**Review by swetha-charles — CHANGES_REQUESTED** [review]

File: components/layout/NavigationMenu.tsx (hunk 1 above)
Comment (PatriciaRosalis) [reply components/layout/NavigationMenu.tsx]: Totally makes sense! Deleted the router dependency :)

**Comment by PatriciaRosalis** [comment]
Thanks, @eleanorreem !! What a great review! 😸

**Review by swetha-charles — APPROVED** [review]

**Pull request merged**